python/sglang/srt/server_args.py

```diff
@@ -131,6 +134,29 @@ def __post_init__(self):
         if self.random_seed is None:
             self.random_seed = random.randint(0, 1 << 30)
 
+        # Deprecation warnings
+        if self.disable_flashinfer:
+            logger.warning(
+                "The option '--disable-flashinfer' will be deprecated in the next release. "
+                "Please use '--attention-backend triton' instead."
+            )
+        if self.disable_flashinfer_sampling:
+            logger.warning(
+                "The option '--disable-flashinfer-sampling' will be deprecated in the next release. "
```

Comment:
should this error out instead of warning? since `--disable-flashinfer` is not taking effect anymore. hitting this issue on my end @merrymercy

Reply:
the other way is, if `--disable-flashinfer` is true, we set `--attention-backend triton` implicitly

Reply:
Sorry, it is my bug @ByronHsu. I should set `--attention-backend triton` automatically in this case. I will file a fix.

Reply:
https://github.com/sgl-project/sglang/pull/1389